bundles/core/org.eclipse.smarthome.core.library/src/main/java/org/eclipse/smarthome/core/library/types/PointType.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.smarthome.core.library.types;
+
+import java.math.BigDecimal;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.ComplexType;
+import org.eclipse.smarthome.core.types.PrimitiveType;
+import org.eclipse.smarthome.core.types.State;
+
+/**
+ * This type can be used for items that are dealing with GPS or
+ * location awareness functionality.
+ *
+ * @author Gaël L'hopital
+ *
+ */
+public class PointType implements ComplexType, Command, State {
+
+    public static final double EARTH_GRAVITATIONAL_CONSTANT = 3.986004418e14;
+    public static final double WGS84_a = 6378137; // The equatorial radius of WGS84 ellipsoid (6378137 m).
+
+    private BigDecimal latitude; // in decimal degrees
+    private BigDecimal longitude; // in decimal degrees
+    private BigDecimal altitude = BigDecimal.ZERO; // in decimal meters
+
+    // constants for the constituents
+    static final public String KEY_LATITUDE = "lat";
+    static final public String KEY_LONGITUDE = "long";
+    static final public String KEY_ALTITUDE = "alt";
+
+    private static final BigDecimal circle = new BigDecimal(360);
+    private static final BigDecimal flat = new BigDecimal(180);
+    private static final BigDecimal right = new BigDecimal(90);
+
+    public static final PointType EMPTY = new PointType(new DecimalType(0), new DecimalType(0));
```

Comment:
What is this good for? 0,0 is actually a real physical location on earth, so why call it EMPTY?